src/main/resources/application-local.yml

```
@@ -4,24 +4,21 @@ spring:
       on-profile: local
 
   datasource:
-    url: jdbc:h2:tcp://localhost/~/projects/ddingdong/ddingdong
-    driver-class-name: org.h2.Driver
-    username: sa
-    password:
+    url: jdbc:mysql://localhost:3307/ddingdong_local_db
+    driver-class-name: com.mysql.cj.jdbc.Driver
+    username: root
+    password: 1234
 
   jpa:
     hibernate:
       ddl-auto: create
-    show-sql: true
     properties:
       hibernate:
         format_sql: true
+        dialect: org.hibernate.dialect.MySQL5InnoDBDialect
```

p3) 단순 궁금증인데요, 혹시 5버전 사용하시는 이유가 있을까요?

---

```suggestion
        dialect: org.hibernate.dialect.MySQLDialect
```
사실 위에 버전으로 했는데, 계속 entitymanagefactory가 없다는 오류가 떠가지고 보니까, 이 버전이 잘못된것 같아서
prod버전을 사용했는데 8 버전으로 바꾸는 게 좋을까요?

---

넵! 8버전으로 가면 될 것 같아요~

```suggestion
        dialect: org.hibernate.dialect.MySQL8InnoDBDialect
```